deploy/addons/awsecr-creds/awsecr-creds-rc.yaml

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
Not sure if its exactly needed on these yamls - but the K8s Copyright Header is missing here